ArduPlane/quadplane.cpp

```diff
@@ -2605,14 +2605,18 @@ void QuadPlane::vtol_position_controller(void)
                                                   2*position2_dist_threshold_m + stopping_distance_m(rel_groundspeed_sq));
 
                 target_speed_ne_ms = diff_wp_norm * approach_speed_ms;
-                have_target_yaw = true;
-
-                // adjust target yaw angle for wind. We calculate yaw based on the target speed
-                // we want assuming no speed scaling due to direction
+                
+                // adjust target yaw angle into the apparent wind
                 const Vector2f wind_ms = plane.ahrs.wind_estimate().xy();
-                const float gnd_speed_ms = plane.ahrs.groundspeed();
-                Vector2f target_speed_xy = landing_velocity_ne_ms + diff_wp_norm * gnd_speed_ms - wind_ms;
-                target_yaw_deg = degrees(target_speed_xy.angle());
+                const Vector2f airspeed_ne_ms = plane.ahrs.groundspeed_vector() - wind_ms;
+                if (airspeed_ne_ms.length_squared() < 1) {
```

Comment:
I had a look into the thresholds we use elsewhere. Copter also uses 1m/s here:

https://github.com/ArduPilot/ardupilot/blob/90309858cb6c00868a7ac4f5cf74a8a500a0805f/ArduCopter/autoyaw.cpp#L22

When in WP Nav copter uses 5% of the set max speed. Here:

https://github.com/ArduPilot/ardupilot/blob/90309858cb6c00868a7ac4f5cf74a8a500a0805f/libraries/AC_AttitudeControl/AC_PosControl.cpp#L1636

By default on quadplane that would be 5m/s, so 5% would be 0.25m/s. However, that is using the targets so should be less prone to noise than the measured ground speed.